src/ripple/rpc/handlers/Version.h

```diff
@@ -0,0 +1,63 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
```

Comment:
Simply having the copyright in every file gives us 98% of the value. Not sure if its worth chasing after the remaining 2%.